src/openrct2/ride/coaster/ClassicWoodenRollerCoaster.cpp

```diff
@@ -0,0 +1,1953 @@
+        case TrackElemType::Down25ToLeftBank:
+            return classic_wooden_rc_track_25_deg_down_to_left_bank;
+        case TrackElemType::Down25ToRightBank:
+            return classic_wooden_rc_track_25_deg_down_to_right_bank;
+        case TrackElemType::LeftBank:
+            return classic_wooden_rc_track_left_bank;
+        case TrackElemType::RightBank:
+            return classic_wooden_rc_track_right_bank;
+        case TrackElemType::LeftBankedQuarterTurn3Tiles:
+            return classic_wooden_rc_track_left_quarter_turn_3_bank;
+        case TrackElemType::RightBankedQuarterTurn3Tiles:
+            return classic_wooden_rc_track_right_quarter_turn_3_bank;
+        case TrackElemType::LeftEighthBankToDiag:
+            return classic_wooden_rc_track_left_eighth_bank_to_diag;
+        case TrackElemType::RightEighthBankToDiag:
+            return classic_wooden_rc_track_right_eighth_bank_to_diag;
+        case TrackElemType::LeftEighthBankToOrthogonal:
+            return classic_wooden_rc_track_left_eighth_bank_to_orthogonal;
+        case TrackElemType::RightEighthBankToOrthogonal:
+            return classic_wooden_rc_track_right_eighth_bank_to_orthogonal;
+        case TrackElemType::DiagFlatToLeftBank:
+            return classic_wooden_rc_track_diag_flat_to_left_bank;
+        case TrackElemType::DiagFlatToRightBank:
+            return classic_wooden_rc_track_diag_flat_to_right_bank;
+        case TrackElemType::DiagLeftBankToFlat:
+            return classic_wooden_rc_track_diag_left_bank_to_flat;
+        case TrackElemType::DiagRightBankToFlat:
+            return classic_wooden_rc_track_diag_right_bank_to_flat;
+        case TrackElemType::DiagLeftBankToUp25:
+            return classic_wooden_rc_track_diag_left_bank_to_25_deg_up;
+        case TrackElemType::DiagRightBankToUp25:
+            return classic_wooden_rc_track_diag_right_bank_to_25_deg_up;
+        case TrackElemType::DiagUp25ToLeftBank:
+            return classic_wooden_rc_track_diag_25_deg_up_to_left_bank;
+        case TrackElemType::DiagUp25ToRightBank:
+            return classic_wooden_rc_track_diag_25_deg_up_to_right_bank;
+        case TrackElemType::DiagLeftBankToDown25:
+            return classic_wooden_rc_track_diag_left_bank_to_25_deg_down;
+        case TrackElemType::DiagRightBankToDown25:
+            return classic_wooden_rc_track_diag_right_bank_to_25_deg_down;
+        case TrackElemType::DiagDown25ToLeftBank:
+            return classic_wooden_rc_track_diag_25_deg_down_to_left_bank;
+        case TrackElemType::DiagDown25ToRightBank:
+            return classic_wooden_rc_track_diag_25_deg_down_to_right_bank;
+        case TrackElemType::DiagLeftBank:
+            return classic_wooden_rc_track_diag_left_bank;
+        case TrackElemType::DiagRightBank:
+            return classic_wooden_rc_track_diag_right_bank;
+    }
+
+    return get_track_paint_function_classic_wooden_rc_fallback(trackType);
```

Comment:
Couldn't thisone just work with `return get_track_paint_function_wooden_rc(trackType);`?

Reply:
This is something @Gymnasiast added, I didn't change it.

Reply:
I'm sure it'll be fine, but I'll try something in the morning

Reply:
Yea you can dismiss this comment.